exporters/otlp/src/otlp_grpc_client.cc

```diff
@@ -333,7 +333,16 @@ std::shared_ptr<grpc::Channel> OtlpGrpcClient::MakeChannel(const OtlpGrpcClientO
   }
 
   std::shared_ptr<grpc::Channel> channel;
-  std::string grpc_target = url.host_ + ":" + std::to_string(static_cast<int>(url.port_));
+  std::string grpc_target;
+
+  if (url.scheme_ == "unix")
```

Comment:
Could you please add a test for unix socket?

Reply:
Also add some example on the unix scheme in the comment to help understanding?